MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json

```diff
@@ -12,6 +12,10 @@
   "categories": {
     "OWASP_MASVS_L2": [
       "MSTG_AUTH_8"
+    ],
+    "GDPR_Chapter_IV": [
+      "Article 32 - Security of processing",
+      "Article 25 - protection by design and by default"
```

Comment:
You should add the GDPR category in scan_categories in RE and just mention the key here.